models/org.go

```diff
@@ -298,12 +296,16 @@ func GetOwnedOrgsByUserIDDesc(userID int64, desc string) ([]*User, error) {
 // GetOrgUsersByUserID returns all organization-user relations by user ID.
 func GetOrgUsersByUserID(uid int64, all bool) ([]*OrgUser, error) {
 	ous := make([]*OrgUser, 0, 10)
-	sess := x.Where("uid=?", uid)
+	sess := x.
+		Join("LEFT", "user", `"org_user".org_id="user".id`).
+		Where("uid=?", uid)
```

Comment:
It seem correct, Here we want organizations ordered by name, and in GOGS, organization are just a simple user.